.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'development'
+      - 'master'
+      - 'release/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.1.0
+      - name: Setup JDK
+        uses: actions/setup-java@v3.5.1
+        with:
+          java-version: 11
+          distribution: temurin
+      - name: Setup Android SDK
+        uses: android-actions/setup-android@v2.0.2
+      - name: Assemble
+        run: ./gradlew --no-daemon assembleDebug
```

Comment:
Since you have a container to yourself I think the daemon is fine.

Reply:
Ah okay good to know I got that from a Medium article I was following. Will remove it in the Android Lint PR

Reply:
I added it back cause we were getting flacky builds